pkg/nfd-master/nfd-master.go

```diff
@@ -891,22 +889,18 @@ func authorizeClient(c context.Context, checkNodeName bool, nodeName string) err
 		// Check that the node name matches the CN from the TLS cert
 		client, ok := peer.FromContext(c)
 		if !ok {
-			klog.Errorf("gRPC request error: failed to get peer (client)")
 			return fmt.Errorf("failed to get peer (client)")
 		}
 		tlsAuth, ok := client.AuthInfo.(credentials.TLSInfo)
 		if !ok {
-			klog.Errorf("gRPC request error: incorrect client credentials from '%v'", client.Addr)
 			return fmt.Errorf("incorrect client credentials")
 		}
 		if len(tlsAuth.State.VerifiedChains) == 0 || len(tlsAuth.State.VerifiedChains[0]) == 0 {
-			klog.Errorf("gRPC request error: client certificate verification for '%v' failed", client.Addr)
 			return fmt.Errorf("client certificate verification failed")
 		}
 
 		err := verifyNodeName(tlsAuth.State.VerifiedChains[0][0], nodeName)
 		if err != nil {
-			klog.Errorf("gRPC request error: authorization for %v failed: %v", client.Addr, err)
```

Comment:
are we missing replacement `klog.ErrorS()` here ?

Reply:
Hmm, thanks for pointing this out. I wonder what I was thinking with dropping all of these 🤨 Probably cutting corners in the name of duplication (because we get the error on the client side, too). But I think we should not change this -> let me fix this

Reply:
This is now resolved by logging the returned error in `SetLabels()`